docker/Dockerfile

```diff
@@ -1,41 +1,29 @@
-FROM registry.service.dsd.io/ruby:2.6.3
+FROM ruby:2.6.3-alpine3.8
 
-# add official nodejs repo
-RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
-        echo 'deb https://deb.nodesource.com/node jessie main' > /etc/apt/sources.list.d/nodesource.list
+# Install pre-requisites for building nokogiri & pg gems, as well
+# as the packages for converting and generating PDF's
+RUN apk --update add --virtual build_deps \
+    build-base ruby-dev libc-dev linux-headers \
+    libxml2-dev libxslt-dev \
+    postgresql-dev pdftk runit nodejs libreoffice redis git
 
-# install runit, nodejs and libreoffice (used by ADP to generate PDFs)
-# pdftk is used to fill in PDF forms for disc evidence download
-RUN apt-get update && apt-get install -y pdftk runit nodejs libreoffice redis-server && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* && rm -fr *Release* *Sources* *Packages* && \
-    truncate -s 0 /var/log/*log
+RUN addgroup -g 1000 -S appgroup && \
+    adduser -u 1000 -S appuser -G appgroup
 
-# SSH proxy settings
-ENV SSH_AUTH_SOCK /tmp/ssh-auth
-ENV SSH_AUTH_PROXY_PORT 1234
+WORKDIR /app
```

Comment:
I have gone with using `/usr/src/app` in the dummy app as it is a folder that already exists and seems to be a [linux convention](http://forums.devshed.com/linux-help-33/difference-usr-src-usr-local-src-60913.html) for such things

Reply:
so 
```
RUN mkdir -p /usr/src/app
WORKDIR /usr/src/app
RUN mkdir -p /usr/src/app/tmp
```